### PR TITLE
Remove sleep in webhook integration tests

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
Fixes flakyness of webhook tests replacing the sleep with a wait for func.

### CODE REVIEW

**Review by ddonahue007 — DISMISSED** [review]
lgtm

File: tests/integration/event_source_webhook/test_webhook_source.py [hunk 1]
```diff
@@ -8,6 +8,20 @@
 from ..utils import CLIRunner, TESTS_PATH
 
 
+def wait_for_events(proc: subprocess.Popen, timeout: float = 5.0):
```
Comment (bzwei) [inline tests/integration/event_source_webhook/test_webhook_source.py:11]: why can't you use the same method from utils?

Comment (Alex-Izquierdo) [reply tests/integration/event_source_webhook/test_webhook_source.py:11]: It's a mistake, it's should not be there.

**Review by bzwei — APPROVED** [review]

**Review by ttuffin — APPROVED** [review]

**Review by ddonahue007 — APPROVED** [review]
lgtm

**Pull request merged**